code/game/machinery/vending.dm

```diff
@@ -1003,21 +1008,23 @@
 
 /obj/machinery/vending/security
 	name = "\improper SecTech"
-	desc = "A security equipment vendor."
-	product_ads = "Crack capitalist skulls!;Beat some heads in!;Don't forget - harm is good!;Your weapons are right here.;Handcuffs!;Freeze, scumbag!;Don't tase me bro!;Tase them, bro.;Why not have a donut?"
+	desc = "A security equipment vendor, with some customary donuts."
+	product_ads = "Defend yourself!;Help others!;All you need for a secure facility right here!;Your weapons are right here.;Handcuffs!;Did you know that 'Freeze, scumbag!' is the most standardised quote broadcasted by this machine!;Try your best to apprehend all criminals and subject them to the law.;Reminder to always use non-lethals first!;Why not have a donut?"
```

Comment:
Should leave in "Freeze Scumbag" alongside the modified version you made cause its funny then.

Reply:
Suggest it and I'll commit it with you as a co-author I'm too tired.

Reply:
Is it not already in there... wut.

Reply:
no its not, you edited the original one

Reply:
Okay, I'll do that.


> no its not, you edited the original one